src/core/p5.Renderer.js

```diff
@@ -298,15 +298,12 @@ p5.Renderer.prototype.text = function(str, x, y, maxWidth, maxHeight) {
       }
     } else {
       // no text-height specified, show warning for BOTTOM / CENTER
-      if (this._textBaseline === constants.BOTTOM) {
-        return console.warn(
-          'textAlign(*, BOTTOM) requires x, y, width and height'
-        );
-      }
-      if (this._textBaseline === constants.CENTER) {
-        return console.warn(
-          'textAlign(*, CENTER) requires x, y, width and height'
-        );
+      if (this._textBaseline === constants.BOTTOM ||
+        this._textBaseline === constants.CENTER) {
+        // use rectHeight as an approximation for text height
+        let rectHeight = p.textSize() * (this._textLeading || 1.25);
```

Comment:
It looks like `this._textLeading` is always initialized here https://github.com/processing/p5.js/blob/4e231d95ca9408252d464cd4395c3244c0566e19/src/core/p5.Renderer.js#L40 and also set when the user calls `textSize()`, so I think it's safe to just use `this._textLeading` here.

If there's a case I'm not considering though, and we need the `|| 1.25`, can we replace 1.25 with `constants._DEFAULT_LEADMULT` like we use here? https://github.com/processing/p5.js/blob/4e231d95ca9408252d464cd4395c3244c0566e19/src/core/p5.Renderer.js#L163